v20.1/cockroachcloud-upgrade-policy.md

```diff
@@ -1,10 +1,91 @@
 ---
-title: Cluster Upgrades
+title: Upgrade Your Cluster
 summary: Learn about the CockroachCloud upgrade policy.
 toc: true
 build_for: [cockroachcloud]
 ---
 
+Now that [CockroachDB v20.1](https://www.cockroachlabs.com/docs/releases/v20.1.0.html) is available, you can upgrade your cluster directly from the CockroachCloud Console. This page walks you through the process.
+
+## Step 1. Select your cluster size
+
+The upgrade process depends on the number of nodes in your cluster. Select whether your cluster has a single node or multiple nodes:
+
+<div class="filters filters-big clearfix">
+  <button class="filter-button" data-scope="single-node">Single-node</button>
+  <button class="filter-button" data-scope="multi-node">Multi-node</button>
+</div>
+
+## Step 2. Understand the process
+
+<section class="filter-content" markdown="1" data-scope="single-node">
+
+When upgrading a single-node cluster, the cluster will be briefly unavailable while the node is stopped and restarted with v20.1.
+
+Once the node has been restarted, you will have approximately 24 hours to monitor your application and, if necessary, rollback to v19.2 from the CockroachCloud Console. Otherwise, the upgrade will be finalized automatically. You'll also have the choice to finalize more quickly. 
+
+Once the upgrade has been finalized, either automatically or manually, certain features and performance improvements in v20.1 will be available. However, it will no longer be possible to roll back to v19.2. If you see unexpected behavior, you will have to reach out to support.
+
+</section>
+<section class="filter-content" markdown="1" data-scope="multi-node">
+
+When upgrading a multi-node cluster, the upgrade happens one node at a time without interrupting the cluster's overall health and availability. This "rolling upgrade" approach is possible due to CockroachDB's [multi-active availability](multi-active-availability.html) design.
+
+Once all nodes are running v20.1, you will have approximately 24 hours to monitor your application and, if necessary, rollback to v19.2 from the CockroachCloud Console. Otherwise, the upgrade will be finalized automatically. You'll also have the choice to finalize more quickly.
+
+Once the upgrade has been finalized, either automatically or manually, certain features and performance improvements in v20.1 will be available. However, it will no longer be possible to roll back to v19.2. If you see unexpected behavior, you will have to reach out to support.
```

Comment:
Link list of features for 20.1?

Reply:
I see it's down below. Perhaps still link to later in same page? Feel free to ignore!

Reply:
Done now, but on new page.